Libs/MRML/Core/vtkMRMLSliceNode.h

```diff
@@ -78,11 +78,27 @@ class VTK_MRML_EXPORT vtkMRMLSliceNode : public vtkMRMLAbstractViewNode
   vtkGetMacro(SliceVisible, int);
   vtkSetMacro(SliceVisible, int);
 
+  ///
+  /// The visibility of the slice edge in the 3DViewer.
+  vtkGetMacro(ThreeDSliceEdgeVisibility, int);
```

Comment:
Using int instead of bool was a limitation in Tcl decades ago. For new Boolean variables we should use bool type.